tests/code_generation/unit/test_function.py

```diff
@@ -252,3 +256,11 @@ def test_function_prefix(self):
         source.prefix = "__declspec(dllexport)"
         result = self.printer.print(source)
         self.assertEqual(result, "__declspec(dllexport) long foo();\n")
+
+    def test_consteval_body(self):
+        source = FunctionInfo.create_default()
+        source.name = "foo"
+        source.return_type = TypeExpression.create_default()
+        source.return_type.details = BasicType.VOID
+        source.body = "if consteval() { foo(); }"
+        self.assertFalse(source.modification.is_consteval)
```

Comment:
This test is broken. In code generation tests you should test generated string, not creation of class.